<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QWidget" name="logWidget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="mainWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="controlWidget" native="true">
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QVBoxLayout" name="takePictureLayout">
           <item>
            <widget class="QPushButton" name="selectButton">
             <property name="font">
              <font>
               <family>Sans</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Take Picture</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="whoLabel">
             <property name="font">
              <font>
               <family>Sans</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Are you ...?</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="buttonLayout">
           <item>
            <widget class="QPushButton" name="yesButton">
             <property name="font">
              <font>
               <family>Sans</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Yes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="noButton">
             <property name="font">
              <font>
               <family>Sans</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>No</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="indicatorLabel">
           <property name="font">
            <font>
             <family>Sans</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>If Not, Who are you?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="selectComboBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="trainButton">
           <property name="text">
            <string>Train</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
